mp/src/game/server/neo/neo_player.cpp

```diff
@@ -1960,6 +1972,21 @@ void CNEO_Player::SetPlayerTeamModel( void )
 #endif
 }
 
+// Input to set the player's model (not skin)
+void CNEO_Player::InputSetPlayerModel( inputdata_t & inputdata )
+{
+	const char* modelpath = inputdata.value.String();
+
+	if (modelpath && *modelpath)
```

Comment:
```suggestion
	if (modelpath)
```

Reply:
This stops the game from crashing when the input is sent and there is no input parameter specified